openedx/adg/lms/webinars/admin.py

```diff
@@ -119,7 +134,7 @@ def save_related(self, request, form, formsets, change):
 
     def get_queryset(self, request):
         qs = super(WebinarAdmin, self).get_queryset(request)
-        return qs.filter(~Q(status=Webinar.CANCELLED))
+        return qs.filter(is_cancelled=False)
```

Comment:
We can use the method that we will add in queryset `queryset.cancelled_webinars()`

Reply:
Same for all usages.